.vscode/tasks.json

```diff
@@ -34,7 +34,7 @@
         {
             "label": "test",
             "type": "shell",
-            "command": "cd src && ./install_coverage_dependencies.sh && ./coverage.sh",
+            "command": "cd src && ./test.sh",
```

Comment:
- Is this coverage.sh renamed as test.sh? 
- install_coverage_dependencies.sh is removed , but is this gvor installed with some other scripts?

Reply:
Yes coverage is renamed as test.sh.  I removed from the test task in VSCode the functionality to generate coverage results by default since normally this is not the intention. Normally people running this task expect only that their unit tests are run. Instead they get may coverage files they do not expect or understand.

I can also add two different tasks. One task with just testing and one task with testing and coverage.

Reply:
Added a test with coverage task